README.md

```diff
@@ -36,7 +36,7 @@ We
 
 - created an actual [addon operator we deemed as straight-forward](https://github.com/kubernetes-sigs/cluster-addons/tree/master/coredns), so we have actual code to look
 - wrote an [installer library](https://github.com/kubernetes-sigs/cluster-addons/tree/master/installer) to install addons into the cluster
-- added support for [addon operators to kubebuilder](https://github.com/kubernetes-sigs/kubebuilder/tree/master/plugins)
+- added support for [addon operators to kubebuilder](https://github.com/kubernetes-sigs/kubebuilder/tree/master/pkg/plugins)
```

Comment:
Or should I update here with "https://github.com/kubernetes-sigs/kubebuilder/tree/master/pkg/plugins/golang/declarative/v1" ?
Because that is more clearly, I think.

Reply:
Yes, that makes sense!

Reply:
Ok!I'll update here.

Reply:
done.
thank you to quick comment! :smile: